### ISSUE TITLE
[라이프로그_활동] 교차검증 시 accuracy 값이 하나의 값으로 수렴하는 문제 발생 

### ISSUE THREAD

**Issue by yonghyeun** [yonghyeun/Dementia_Analysis#4]
```python
params = {'DROPOUT_RATE':0.4,
        'START_NODE':32,
        'HIDDEN_NODE':16,
        'LAYERS':5}

optimizers = {'loss':'BinaryCrossentropy',
        'optimizer':'adam',
        'metrics':'accuracy'}

REPEATE = 10

fig, ax = plt.subplots(ncols = 5, nrows = 2, figsize = (20,10))
ax = ax.flatten()

for num in range(REPEATE):
        x_train, x_test,y_train,y_test = train_test_split(X,Y,test_size = 0.2, random_state = 42 + num,
                                                                stratify = Y)
        
        model = DNN(**params)
        model.compile(**optimizers)
        
        history = model.fit(x_train,y_train,
                        validation_data = (x_test,y_test),
                        epochs = 9999,
                        callbacks = [early_stopping],
                        verbose = 0)
        ax[num].plot(history.history['accuracy'], label = 'train_accuracy', color = 'orange')
        ax[num].plot(history.history['val_accuracy'], label = 'val_accuracy', color = 'blue')  
        
        ax[0].set_ylabel('accuracy')
        ax[5].set_ylabel('accuracy')
        ax[num].legend(loc = 'lower right')
        score = round(model.evaluate(x_test,y_test,verbose = 0)[-1],8)
        ax[num].set_title(f'{num} FOLD SET :{score}',fontsize = 10)
        

```
![b9dbabd1-1b6d-4cc7-80f1-88ecdbf9a035](https://user-images.githubusercontent.com/123540354/234811642-d03515b8-0d0e-40f5-81f5-f4dbe037852c.png)

해당 코드에서 교차검증 시 테스트 데이터가 바뀜에도 불구하고 대부분의 accuracy 값이 비슷한 값에 수렴하는 문제 발생 

random_state 값을 계속 바꿔주고 있는데 잘 바뀌지 않나?

**Comment by yonghyeun** [comment]
지금 보니 
```python
    x_train,x_test,y_train,y_test = train_test_split(X,Y,random_state = 42 + i, stratify = Y)
```

부분에서 stratify 를 설정해주지 않으면 점수가 들쑥 날쑥 하지만 stratify 를 지정해주니 점수들이 모두 동일하게 나오는 것으로 보임 

부족한 데이터의 갯수에서 생기는 문제 같음 ..
